### DISCUSSION TITLE
Howto run CustomTkinter App in Kiosk Mode on Rasberry Pi 4

### DISCUSSION THREAD

**Discussion by chess-levin** [TomSchimansky/CustomTkinter#2321] [General]
Hi @TomSchimansky ,

I've build a vlc frontend app with your great lib CustomTkinter. It was my 1st bigger Python project and it worked out well.

Now I would like to run this app on a raspi 4 in kiosk mode. The raspi should start directly into kiosk mode and show my vlc-frontend app in fullscreen mode - no windows, just the content pane of my app.

Is this possible and how is it done?

Thank you for your support

**Comment by Akascape** [comment]
@chess-levin 
If it's possible with tkinter, then it's also possible with customtkinter. 

The aproach will be same, check these resources:

https://stackoverflow.com/questions/47856817/tkinter-canvas-based-kiosk-like-program-for-raspberry-pi

and 

https://brod.co/articles/run-a-tkinter-app-in-kiosk-mode-on-a-raspberry-pi

**Comment by chess-levin** [comment]
Thanks. I'm going to try it.

**Comment by chess-levin** [comment]
Setting app-properties as described in this [post](https://stackoverflow.com/questions/47856817/tkinter-canvas-based-kiosk-like-program-for-raspberry-pi) works perfect! Thx @Akascape

But I don't get my app running with an autostart-configuration. My RaspiOS is configured to start without login - my default user is alredy logged in when the desktop is up.

I wrote this little script to set the python3 venv and finally start my app:

```
#!/bin/bash
cd ~/showerradio
source showerradio_venv/bin/activate
python3 main_oop.py
```

Then I added an entry to my autostart config, but nothing happens when I reboot the raspi. I need some help here. 

```
[Desktop Entry]
Type=Application
Name=Showerradio
Exec=/home/radio/shwoerradio/run.sh
```

I don't even know where I could find a logfile that tells me what the problem is.

Need some help from you folks...

**Reply by Akascape** [reply]
@chess-levin Better ask on stackoverflow.
